.travis.yml

```diff
@@ -43,7 +49,13 @@ before_script:
   - gcc --version
   - arm-none-eabi-gcc --version
   - python3 --version
-
+# Install pycodestyle & autopep8 python package for PEP8 coding style.
+install:
+  # Install code sniffer.
+  - pip install pycodestyle
```

Comment:
Try doing `pip3 install pycodestyle` here. I don't think the language setting isn't having influence over this because only one language (c in this case) can be selected at a time.

Reply:
@tannewt : i already tried with pip3.

travis error:- 

$ pip3 install pycodestyle
pyenv: pip3: command not found
The `pip3' command exists in these Python versions:
 3.5
 3.5.3

Reply:
Ah bummer! I might have to do with the installation order of python versus the pip command.